doc/src/arch/reference.rst

```diff
@@ -1454,7 +1462,7 @@ The following tags are common to all <pb_type> tags:
             </mode>
         </pb_type>
 
-    specifies the ``lut`` pb_type can be used as either a single 6-input LUT, or as two 5-input LUTs (but not both).
+    specifies the ``lut`` pb_type can be used as either a single 6-input LUT, or as two 5-input LUTs (but not both). Mode ``lut5x2`` is set to be unpackable, and therefore will not be considered for mapping resources during VPR packing stage. Mode ``lut6`` is by default set to be packable, where VPR packer will try to map resources.
```

Comment:
I'm not sure putting packable=false in the example is a great idea, since it is really a power-user debugging option and hence not typical usage.

I could anticipate a new user thinking: 'Why would you define this lut5x2 mode but make it unpackable?'. Probably better to leave it out here, and make a packer debugging tutorial like @mithro suggested, which can be cross-refed here (e.g. ``.. seealso::`).

Reply:
Indeed, this is a feature for power users to debug the packer and architecture file.
I think a tutorial for packer debugging where bugs in architecture file can be quickly spotted by using the ``disable_packing`` switch and the verbose output of packer.